lib/merge.test.js

```diff
@@ -0,0 +1,144 @@
+            },
+          ],
+        },
+      ],
+      config.predictions
+    );
+    expect(merge([], {})).toEqual([]);
+    expect(merge(data, predictions)).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "Algen": "",
+          "BSL": "B336",
+          "BadName": "Radfahrerwiese",
+          "BadestelleLink": "\\"Radfahrerwiese(Link zur Badestelle Radfahrerwiese)\\":/lageso/gesundheit/gesundheitsschutz/badegewaesser/badestellen/artikel.344357.php",
+          "Bezirk": "Steglitz-Zehlendorf",
+          "Dat": "15.09.2020",
+          "Eco": "<15",
+          "Ente": "<15",
+          "Farb_ID": "1",
+          "Farbe": "gruen.jpg",
+          "Latitude": "52.45861200",
+          "Longitude": "13.18963400",
+          "PDFLink": "\\"Alle Probeentnamen der aktuellen Saison fÿr diese Badestelle(Link zur PDF-Datei mit allen Probeentnamen der aktuellen Saison)\\":http://ftp.berlinonline.de/lageso/baden/bad35.pdf",
+          "Profil": "Unterhavel",
+          "ProfilLink": "\\"Unterhavel - Radfahrerwiese(Link zum Badegewÿsserprofil Unterhavel Radfahrerwiese)\\":/lageso/gesundheit/gesundheitsschutz/badegewaesser/badegewaesserprofile/artikel.339135.php",
+          "PrognoseLink": "",
+          "RSS_Name": "Radfahrerwiese / Unterhavel",
+          "Sicht": "110",
+          "Temp": "20,2",
+          "Wasserqualitaet": "1",
+          "cb": "<300",
+          "id": "344357",
+          "id2": "339135",
+        },
+        Object {
+          "Algen": "",
+          "BSL": "B331",
+          "BadName": "Lieper Bucht",
+          "BadestelleLink": "\\"Lieper Bucht(Link zur Badestelle Lieper Bucht)\\":/lageso/gesundheit/gesundheitsschutz/badegewaesser/badestellen/artikel.344358.php",
+          "Bezirk": "Steglitz-Zehlendorf",
+          "Dat": "15.09.2020",
+          "Eco": "<15",
+          "Ente": "<15",
+          "Farb_ID": "12",
+          "Farbe": "gruen_a_prog.jpg",
+          "Latitude": "52.46933200",
+          "Longitude": "13.19672100",
+          "PDFLink": "\\"Alle Probeentnamen der aktuellen Saison fÿr diese Badestelle(Link zur PDF-Datei mit allen Probeentnamen der aktuellen Saison)\\":http://ftp.berlinonline.de/lageso/baden/bad34.pdf",
+          "Profil": "Unterhavel",
+          "ProfilLink": "\\"Unterhavel - Lieper Bucht(Link zum Badegewÿsserprofil Unterhavel Lieper Bucht)\\":/lageso/gesundheit/gesundheitsschutz/badegewaesser/badegewaesserprofile/artikel.339132.php",
+          "PrognoseLink": "\\"Erklÿrung zum Frÿhwarnsystem(Link zur Erklÿrung zum Frÿhwarnsystem)\\":http://www.berlin.de/lageso/gesundheit/gesundheitsschutz/badegewaesser/fruehwarnsystem-unterhavel/index.php /",
+          "RSS_Name": "Lieper Bucht / Unterhavel",
+          "Sicht": "90",
+          "Temp": "20,3",
+          "Wasserqualitaet": "12",
+          "cb": "<300",
+          "id": "344358",
+          "id2": "339132",
+        },
+      ]
+    `);
+  });
+
+  test("merge lageso is worse then prediction", () => {
```

Comment:
@sebastian-meier This is still pretty vague. I don't really get the logic of the merge and the numbers when what is what. Any improvements you could give me from the top of your head?

Reply:
it's merge.test.js line 92 following